app/(modals)/organizations/[id].tsx

```diff
@@ -22,19 +23,23 @@ export default function AssoIdPage() {
   return (
     <PageContainer>
       <Header title="Profil" leftIcon="back" rightIcon="close" />
-      <Profile
-        avatar={data?.organization.logo_url}
-        handleRefresh={handleRefresh}
-        isRefreshing={isRefreshing}
-        posts={posts}
-        postsAreLoading={postsAreLoading}
-        setSize={setSize}
-        size={size}
-        socials={data?.organization}
-        members={data?.members}
-        subtitle={data?.organization.name}
-        title={data?.organization.short_name}
-      />
+      {!data || isLoading ? (
+        <ActivityIndicator />
+      ) : (
+        <Profile
+          avatar={data?.organization.logo_url}
+          handleRefresh={handleRefresh}
+          isRefreshing={isRefreshing}
+          posts={posts}
+          postsAreLoading={postsAreLoading}
+          setSize={setSize}
+          size={size}
```

Comment:
size will always have a value ?

Reply:
yes, it's for pagination and infinite scrolling, same as in posts